nix/download.tt

```diff
@@ -4,23 +4,30 @@
 
 <p>The latest and recommended version of Nix is
 <strong>[%latestNixVersion%]</strong>. The quickest way to install it
-on Linux and Mac OS X is to run the following in a shell (as a user
+on Linux and macOS is to run the following in a shell (as a user
 other than <tt>root</tt>):
 
-<pre><span class="muted">$</span> curl <a href="[%root%]nix/install">https://nixos.org/nix/install</a> | sh</pre>
+<pre>
+<span class="command">curl -o install-nix-[%latestNixVersion%] https://nixos.org/nix/install</span>
+<span class="command">echo "[%nixInstallHashsha256%]  install-nix-[%latestNixVersion%]" | (shasum -p 256 || sha256sum -c) 2> /dev/null</span>
+<span class="command">sh install-nix-[%latestNixVersion%]</span>
+</pre>
 
-Make sure to follow the instructions output by the script.
+<p>Make sure to follow the instructions output by the script.</p>
 
-You may want to verify the integrity of the installation script using GPG:
+<p>You may want to verify the integrity of the installation script
+using GPG:</p>
 
 <pre>
-<span class="muted">$</span> curl -O <a href="[%root%]nix/install">https://nixos.org/nix/install</a>
-<span class="muted">$</span> curl -O <a href="[%root%]nix/install.sig">https://nixos.org/nix/install.sig</a>
-<span class="muted">$</span> gpg2 --verify ./install.sig
-<span class="muted">$</span> sh ./install
+<span class="command">curl -o install-nix-[%latestNixVersion%] https://nixos.org/nix/install</span>
+<span class="command">curl -o install-nix-[%latestNixVersion%].sig https://nixos.org/nix/install.sig</span>
+<span class="command">gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE</span>
+<span class="command">gpg2 --verify ./install-nix-[%latestNixVersion%].sig</span>
```

Comment:
Hmm... I think how to use gpg is out of scope of this. Do you have any specific suggestions on how to handle the problem you've described? I guess what I'm hoping is if someone doesn't know how to use gpg, they won't muck about with the provided commands 😬.